### DISCUSSION TITLE
iPhone: Some clients do connect, others don´t

### DISCUSSION THREAD

**Discussion by Khymick** [ampache/ampache#3495] [Q&A]
Hi there,

I have an ampache server 5.5.7 running on an ubuntu server 22.04.
Now I want to use an subsonic client from my iPhone. At the moment, I have installed:
- Amperfy (working)
- play:sub (not working)
- AVSub (not working)

Strangely, at least I can tell that play:sub did work in the past. I did reset my iPhone and somehow, since then, it stopped working. I use the same url as for amperfy. When I enter my url, user and password in play:sub, I get immediately a message saying that the "subsonic-server under <url> could not be found". 

As it works with Amperfy, I guess it´s a problem of the client. But how to find out what´s wrong? On the iPhone, I do not have any further logs that could help me. Is there any way to see on the ampache server what is happening?

I would really appreciate to find out what is the cause... Can somebody help me please?

**Comment by Khymick** [comment]
I came across something: In /var/log/apache2/ampache.access.log, I see the following entry when I try to connect from play:sub

ip - username [29/Mar/2023:19:54:23 +0000] "POST /rest/ping.view HTTP/1.1" 200 5214

But I don´t know what this means. Anyone who can help here?

**Reply by lachlan-00** [reply]
it could be that the client is looking for a certain data in the response.
some subsonic clients are really strict. and it could also be a bug depending on the format (xml or json) the client wants.

there are a few iphone people here but i'm not one of them.

can you connect to the demo site okay? https://github.com/ampache/ampache/wiki/demo-server
